### DISCUSSION TITLE
Can we please get a toggle for ship combat sounds and combat music?

### DISCUSSION THREAD

**Discussion by theboy6545** [BeeStation/NSV13#1172] [Ideas]
I don't need to hear the fucking pop pop pop of the 50cals as a chef, and most of the combat music is pretty shit, and it gets kinda repetitive after the 14th shift, so can we please get a toggle for them?

**Comment by Bokkiewokkie** [comment]
Or just a way to turn up/down the volume. I'd like to hear my coworkers scream during combat instead of every sound being completely blocked out by gunfire.
